doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
```

Comment:
Yes, I agree, and that would be good, right? Just being minor? If we wanted, it could be deprecated in a PR towards 6.x, and then removed in 7.x... If I understand correctly.